persistence/src/test/java/com/epam/esm/repository/CertificateRepositoryTest.java

```diff
@@ -0,0 +1,114 @@
+package com.epam.esm.repository;
+
+import com.epam.esm.config.TestConfig;
+import com.epam.esm.entity.Certificate;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {TestConfig.class})
+@ActiveProfiles("dev")
+@Transactional
+class CertificateRepositoryTest {
+
+    private static final long TEST_CERTIFICATE_ID = 5L;
+    public static final long EXISTING_CERTIFICATE_ID = 4L;
+
+    private static final String TEST_CERTIFICATE_NAME = "test";
+    public static final String EXISTING_CERTIFICATE_NAME = "The fourth";
+
+    private Certificate TEST_CERTIFICATE;
+    private Certificate EXISTING_CERTIFICATE;
```

Comment:
These are mutable objects, although the variables that referenced them may be final, you can change the internal state of such objects. So use camelCase for all mutable objects and collections to avoid confusion. 
In general, UPPER_CASE is used only for private static primitive or really immutable fields.

Reply:
fixed it